### ISSUE TITLE
6.8.4 does not work

### ISSUE THREAD

**Issue by TomonoriSoejima** [renshuki/elasticsearch-bootstrap#1]
must be darwin string for OS X.

**Comment by TomonoriSoejima** [comment]
This is how I did it.

```
elasticsearch=""
if [ $(echo $1 | cut -c1) -le 6 ];
then
	elasticsearch=elasticsearch-$1.tar.gz
else
        echo it is 7
  	elasticsearch=elasticsearch-$1-darwin-x86_64.tar.gz

fi
```

**Comment by renshuki** [comment]
Thank you sir, will fix it :)

**Comment by renshuki** [comment]
This should be fixed by 5a00e812cc1c8aea4052190bbd89dcc96dbdf01d 😄
